build.gradle

```diff
@@ -64,6 +64,7 @@ subprojects {
                     }
                     pom {
                         name = 'Honeycomb OpenTelemetry Distribution for Java'
+                        description = 'Honeycomb distribution of the OpenTelemetry Java Instrumentation.'
```

Comment:
Certainly it's simpler to have this one description applied to all the sub-projects for this go around. Is it possible to set the POM description individually on sub-projects in the future to describe their differences? For example:

* honeycomb-opentelemetry-javaagent: "The JavaAgent provides auto-instrumentation and a configurable default tracer."
* honeycomb-opentelemetry-sdk: "The SDK for manual instrumentation without the JavaAgent."
* honeycomb-opentelemetry-common: "Functionality shared across components of the Honeycomb OpenTelemetry distribution."